### ISSUE TITLE
[Overview] UX issues in the CWA

### ISSUE THREAD

**Issue by heinezen** [corona-warn-app/cwa-documentation#666]
## Description

Feel free to use this issue to report smaller UX and UI issues from people on social media. UX issues are any problems that users have with understanding the usage of functionality inside the CWA. Examples for this are:

* Users being unable to find an existing functionality (even though it exists in the app)
* Users having issues navigating the app
* Common mistakes/incorrect usage of a feature
* Misunderstandings about the inner workings of a feature
* Confusion about the relation the other Corona apps (luca, CovPass, CovPassCheck)

In short: Anything that deviates from the intended usage of the app.

## Motivation

Usually, these problems are not necessarily the users' and can be improved by us. The more input we get from real users of the app, the better we can judge where improvements should be made.

----
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
Thanks for this issue! 

Disclaimer: I just forward here what Twitter users told me, this does not have to be my own opinion! 

1. Don't move functions inside of the app w/o further notice. Users where confused when this was done with the contact journal, now it was done again with the vaccination certificates. 
There should be a big red arrow pointing to where the feature now is.
2. The legal text that was shown after a tap on the "Certificates" tab in version 2.4, even if the user had already registered vaccination certificates, were confusing for many users. They thought the certificates were deleted.
3. Stop changing the UI from release to release, with each new version since 2.3, the user had to adapt their usage to the new UI.
First the got used to the certificate on the home screen, then the certificate was moved to it's own tab (showing some "Valid to" date nobody understands or concerns) and then, with version 2.5, they are welcomed with big fat QR-Codes and they wonder where they can find the other infos. There were literally users out there who needed an explanation that they now have to tap on the QR-code.
4. Simplify the texts in the app and make them as short and as understandable as possible.
With each new feature, users of the CWA are normally welcomed with an info page with much texts, only one illustrations and many are unmotivated to use the new feature in this way. And the users that don't just quit the app, will probably not read the texts and just press the "Continue" button.
5. Related to CovPass, see this [Tweet](https://twitter.com/s4lt__/status/1418166603892596738?s=21), where a user says that he thinks that CWA should also display a green ✅, similar to CovPass.
